qa/rpc-tests/bip9-softforks.py

```diff
@@ -79,11 +79,7 @@ def generate_blocks(self, number, version, test_blocks = []):
 
     def get_bip9_status(self, key):
```

Comment:
> found another place in the tests which cut & paste the same function.

I'd rather have this duplicate code removed... You change stuff in one branch of the code and later notice you have missed to apply the patch to all other copies of the code.